src/main/java/com/scalyr/integrations/kafka/ScalyrSinkConnectorConfig.java

```diff
@@ -65,9 +65,12 @@
     + BATCH_SEND_SIZE_BYTES_CONFIG + " may not be reached for longer periods of time.";
   public static final String CUSTOM_APP_EVENT_MAPPING_CONFIG = "custom_app_event_mapping";
   private static final String CUSTOM_APP_EVENT_MAPPING_DOC = "JSON config describing how to map custom application nested Kafka messages to Scalyr events." +
-    "  Multiple custom application event mappings can be specified in a JSON list.  Example config JSON:\n"
-    + "[{\"matcher\": { \"attribute\": \"app.name\", \"value\": \"customApp\"},\n" +
-    " \"eventMapping\": { \"message\": \"message\", \"logfile\": \"log.path\", \"serverHost\": \"host.hostname\", \"parser\": \"fields.parser\", \"version\": \"app.version\"} }]";
+    "  Multiple custom application event mappings can be specified in a JSON list and are evaluated in the order specified in the list." +
+    "  Regex is supported for the matcher.value.  Example config JSON:\n" +
```

Comment:
It is also important to document if the Regex is meant to match the entire field value or just a portion of it.  I would suggest we require them to match the entire field value.  Also please double check that you are using the right Regex method that you require matching the entire value.

Reply:
Added docs to clarify this and verified that regex mathod `matches` matches the entire value.